client/src/test/scala/dwolla/cloudflare/RateLimitClientSpec.scala

```diff
@@ -68,7 +68,7 @@ class RateLimitClientSpec extends Specification {
             period = 600,
             action = RateLimitAction(
               mode = "ban",
-              timeout = 60,
+              timeout = Some(60),
               response = Some(RateLimitActionResponse(
                 contentType = "application/json",
                 body = "{\n  \"code\": \"TooManyRequests\",\n  \"message\": \"Your number of requests has exceeded the limit allowed for the last 1 minute. Your client will be allowed to continue making requests after the specified time in the `Retry-After` header.\"\n}"
```

Comment:
Since we know this is JSON, I'd use the circe JSON interpolator and then serialize to JSON to make sure the output is valid. Something like this (haven't compiled it, so it might be slightly off)

```scala
import io.circe.literal._

json"""{
  "code": "TooManyRequests",
  "message": "Your number of requests…"
}""".noSpaces
```

This will all happen at compile-time, so you'll get compiler errors if the JSON string is invalid.